File: src/interactive-simulator/CasaresModel/Person.cs
using System;
using System.Collections;
using System.Collections.Specialized;
using System.Drawing;

namespace CasaresModel
{
	public class Person
	{
		// Class that handles attributes and actions
		// of each person
	
		// Publics
		public double Wage = 0;
		public Company Company = null;
		public double WageTotal = 0;
		public int Color = 0;
		private int _state = 0;  // State in labor market:  1=Occupied, 2=Unemployed, 3=Inactive
		public int State
		{
			get
			{
				return _state;
			}
			set
			{
				if (_state != value)
				{
					_state = value;
					OnStateChanged();
				}
			}
		}
		public event EventHandler StateChanged;
		protected void OnStateChanged()
		{
			if (StateChanged != null)
				StateChanged(this, null);
		}
		public int Seniority = 0;
		public double Expectations = 0;
		public int UnemploymentLength = 0;
		public Person Spouse = null; // reference to its wife or husband
		public int Id;

		// Privates
	
		// Constructor
		public Person(int id)
		{
			Id = id;
		}
	}
}
	

File: src/console-model/Person.py
from random import random

class Person:
	# Class that handles attributes and actions
	# of each person
	
	# Publics
	Wage = 0
	WageTotal = 0
	Color = 0
	State = 0  # State in labor market:  1=Occupied, 2=Unemployed, 3=Inactive
	Seniority = 0
	Expectations = 0
	UnemploymentLength = 0
	Spouse = None # reference to its wife or husband
	
	# Privates
	
	# Constructor
	def __init__ (self):
		Spouse = None
		return
	
